src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2493,7 +2496,7 @@ static BSONArray processConditionVector
       {
         conds.append(nc->condValueList[jx]);
       }
-
+std::cout << "\nprocessConditionVector apiVersion:\n " << apiVersion << std::endl;
```

Comment:
Seems to be code for debugging. I understand it should be removed.

Reply:
Fixed in a0cb1aa